designs/administrator/future-openshift/autoscaler-guidance/index.md

```diff
@@ -0,0 +1,63 @@
+---
+parent: Administrator
+version: future-openshift
+---
+
+# Machine and Cluster Autoscaler guidance
+
+The flow below helps users out by pointing them toward configuring the Cluster Autoscaler if they haven’t already done so by the time they try to create a Machine Autoscaler. It also shows a user whether a MachineSet has a Machine Autoscaler related to it in the MachineSet list.
+
+## Creating and Editing Cluster Autoscaler
+
+Users will likely first encounter OpenShift’s autoscaling feature from within the Machine Autoscalers nav item.
+
+The user clicks “Create Machine Autoscaler” and is directed to the screen below, if a Cluster Autoscaler has not already been configured:
+
+![Autoscaler](img/autoscaler-0.png)
+
+A brief description of what autoscaling is and how Machine Autoscalers are related to the Cluster Autoscaler is included at the top of the creation form, with a link to the current documentation for users who’d like that.
+
+If the user hasn’t already configured the Cluster Autoscaler (which is likely), then an inline info alert prominently suggests that they do so first before continuing.
+
+The user clicks “Create Cluster Autoscaler.” and is directed to the following form
+![Autoscaler](img/autoscaler-1.png)
+
+The Cluster Autoscaler creation form is largely filled out with defaults aside from the GPU count section, which are unchecked by default. The Maximum number of nodes should be the current number of nodes at the time that this page is loaded.
```

Comment:
So both the machine and cluster autoscaler forms are new? Perhaps they could use the Form/YAML switching interaction that lets the user [freely switch between Form and YAML](http://openshift.github.io/openshift-origin-design/conventions/documentation/create.html#creating-from-create-button)? This would be one of the first non-operator resources to do so, but figured I'd mention. 

Otherwise using the as-drawn form with the option for YAML view would be fine too.

Reply:
@itsptk Have we talked with stakeholders recently about when we might want to move other forms with "Edit YAML" (Routes, PVCs, Storage Classes) to use this new pattern? Maybe it's be worth chatting about again if it's been a while.

Reply:
A few weeks ago I talked with Sam about those 3 resources and he mentioned it might be a rather large effort to convert them, which was sort of surprising to me. I am due to revisit the UX design to make sure it is the most ideal, since it's going to be a lot of work to change over to any other switching design (maybe we just go for the split view then?)

I guess I figured if these were new resources, maybe it would be easier to include it from the beginning? That is somewhat of an assumption though. I guess I'd just be interested in discussing the effort of free switching between form/YAML for these new resources when they are implemented.

Reply:
Agreed @itsptk, we should do this from the start. @LPutney is it possible to update to show the form/yaml radio button toggle instead of the edit yaml link in the top right? Just a nit and more of an implementation detail but might help us ensure all new forms added to console get the new behavior.